specification/policyinsights/resource-manager/Microsoft.PolicyInsights/preview/2018-07-01-preview/examples/PolicyEvents_QuerySubscriptionScope.json

```diff
@@ -0,0 +1,80 @@
+{
+  "parameters": {
+    "policyEventsResource": "default",
+    "subscriptionId": "fffedd8f-ffff-fffd-fffd-fffed2f84852",
+    "api-version": "2018-07-01-preview"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "@odata.nextLink": null,
+        "@odata.context": "https://management.azure.com/subscriptions/fff10b27-fff3-fff5-fff8-fffbe01e86a5/providers/Microsoft.PolicyInsights/policyEvents/$metadata#default",
+        "@odata.count": 2,
+        "value": [
```

Comment:
>value [](start = 9, length = 5)

Please also add "complianceState" field to the response. Same in all events json for this version.

Reply:
hmm this should have already been there.
I will update add it where its missing